lib/index.d.ts

```diff
@@ -1,6 +1,8 @@
-import {ResponseObject} from '@hapi/hapi';
-import {Logger} from 'pino';
-import * as stream from 'stream';
+import { ResponseObject } from '@hapi/hapi';
+import Http from 'http';
+import { Logger } from 'pino';
+import QueryString from 'querystring';
+import Stream, * as stream from 'stream';
```

Comment:
clean this up; you are using both methods for the same reference

Reply:
sure